src/plugins/intel_cpu/tests/functional/behavior/ov_plugin/properties.cpp

```diff
@@ -283,4 +284,30 @@ TEST_F(OVClassConfigTestCPU, smoke_PluginSetConfigExecutionModeAndInferencePreci
     expect_inference_precision(bf16_if_can_be_emulated);
 }
 
+TEST_F(OVClassConfigTestCPU, smoke_PluginSetConfigLogLevel) {
+    ov::Core ie;
+    //check default value
+    ov::Any value;
+    ASSERT_NO_THROW(value = ie.get_property("CPU", ov::log::level));
+    ASSERT_EQ(value.as<ov::log::Level>(), ov::log::Level::NO);
+
+    //check set and get
+    const std::vector<ov::log::Level> logLevels = {ov::log::Level::NO,
+                            ov::log::Level::ERR,
+                            ov::log::Level::WARNING,
+                            ov::log::Level::INFO,
+                            ov::log::Level::DEBUG,
+                            ov::log::Level::TRACE};
+
+    auto check_loglevel_property = [&ie, &logLevels, &value](unsigned int i) {
+        ASSERT_NO_THROW(ie.set_property("CPU", ov::log::level(logLevels[i])));
+        ASSERT_NO_THROW(value = ie.get_property("CPU", ov::log::level));
+        ASSERT_EQ(value.as<ov::log::Level>(), logLevels[i]);
+    };
+
+    for (unsigned int i = 0; i < logLevels.size(); i++) {
+        check_loglevel_property(i);
```

Comment:
minor: Do we really need lambda here? Just range-based loop and asserts inside.

Reply:
fixed it, also added check for except message,
updated the CompiledModel::get_property, and test cases. 
please help to continue review , thanks.